build/ChangeTargetFramework.ps1

```diff
@@ -0,0 +1,31 @@
+﻿Param(
+  [string]$TFparam
+)
+
+$newTF = "netstandard2.0;net45;netstandard1.6"
+
+If (![string]::IsNullOrEmpty($TFparam)) 
```

Comment:
You don't need this when you make the parameter default. See my suggestion above.